lute/std/libs/syntax/query.luau

```diff
@@ -0,0 +1,50 @@
+local luau = require("@std/luau")
```

Comment:
I'm guessing you're splitting the codemod PR up? Can we add a selectRequires query as well?